src/main/java/com/prgrms/commandLineApplication/repository/customer/JdbcCustomerRepository.java

```diff
@@ -0,0 +1,87 @@
+package com.prgrms.commandLineApplication.repository.customer;
+
+import com.prgrms.commandLineApplication.customer.Customer;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.BeanPropertySqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+@Repository
+public class JdbcCustomerRepository implements CustomerRepository {
+
+  private static final String NO_EXIST_CUSTOMER = "It doesn't exist";
+  private final NamedParameterJdbcTemplate jdbcTemplate;
+
+  public JdbcCustomerRepository(NamedParameterJdbcTemplate jdbcTemplate) {
+    this.jdbcTemplate = jdbcTemplate;
+  }
+
+  @Override
+  public Customer save(Customer customer) {
+    SqlParameterSource sqlParameterSource = new BeanPropertySqlParameterSource(customer);
+    jdbcTemplate.update("INSERT INTO customers(customer_id, customer_name, email) VALUES(:customerId, :customerName, :email)"
+            , sqlParameterSource);
+    return customer;
+  }
+
+  @Override
+  public UUID update(UUID customerId, String customerName) {
+    jdbcTemplate.update("UPDATE customers SET customer_name = :customerName WHERE customer_id = :customerId ",
+            Map.of("customerId", customerId, "customerName", customerName));
+    return customerId;
+  }
+
+  @Override
+  public void deleteAll() {
+    jdbcTemplate.update("DELETE FROM customers", Map.of());
+  }
```

Comment:
```suggestion
  @Override
  public void deleteAll() {
    jdbcTemplate.update("DELETE FROM customers", Collections.emptyMap());
  }
```

`Map.of`와 `Collections.emptyMap()`의 차이를 알아보는 것도 좋겠네요

Reply:
어떤 차이가 있고 무엇인지 공부를 한 뒤 이 경우에는 Collections.emptyMap()가 더 적절하다는 걸 알고, 적용하였습니다 !
147d66a